SetupDataPkg/Library/ActiveProfileSelectorLibNull/ActiveProfileSelectorLibNull.c

```diff
@@ -0,0 +1,52 @@
+/** @file
+  Library interface to query the active configuration profile.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include <Uefi.h>
+#include <Library/BaseLib.h>
+#include <Library/DebugLib.h>
+#include <Library/PcdLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/MemoryAllocationLib.h>

```

Comment:
memory allocation can probably be removed now

Reply:
Thanks, will remove.